superset/sql_lab.py

```diff
@@ -366,6 +388,9 @@ def execute_sql_statements(  # pylint: disable=too-many-arguments, too-many-loca
         # only asynchronous queries
         stats_logger.timing("sqllab.query.time_pending", now_as_float() - start_time)
 
+    if not hasattr(g, "user"):
+        g.user = security_manager.find_user(username=user_name)
```

Comment:
SQL Lab doesn't have the `g.user` context, since it might run in Celery workers.